soccer/src/soccer/radio/packet_convert.cpp

```diff
@@ -1,15 +1,40 @@
 #include "packet_convert.hpp"
 
-#include <rj_geometry/util.hpp>
 #include <rj_common/status.hpp>
 #include <rj_common/time.hpp>
+#include <rj_geometry/util.hpp>
 
+#include "control/motion_setpoint.hpp"
 #include "robot_intent.hpp"
 #include "robot_status.hpp"
-#include "motion/motion_setpoint.hpp"
 
-// TODO(#1583): Make this a real ROS parameter
-constexpr float kMaxKickSpeed = 7.0;
+// TODO(#1583): Make these ROS parameters and move them to a central location
+constexpr double kMaxKickSpeed = 7.5;
```

Comment:
Edit: This is more of a general thought that this section makes me think of.

Also will the gameplay side in python be able to access such parameters if they are made to be ROS parameters in a central location? It would be really nice to have everything completely unified, although this brings up some slight implications for the WorldState only design if the gameplay code queried parameters via rclpy, should or are we bundling constants into the WorldState in some way?

Edit: We have bundled all the field constants into the worldstate, but should constants about the robot etc be bundled in there as well?